cloud/pkg/edgecontroller/controller/upstream.go

```diff
@@ -257,7 +262,8 @@ func (uc *UpstreamController) updateRuleStatus() {
 				errSlice := make([]string, 0)
 				rule.Status.Errors = append(errSlice, content.Error.Detail)
 			}
-			newStatus := &rulesv1.RuleStatus{}
+			newStatus := &RuleStatus{}
+			newStatus.Status = rule.Status
```

Comment:
If we always patch the empty status, the count will always 0. We should patch the rule's real status.